_posts/2021-05-03-a-gentle-introduction-to-iceberg.md

```diff
@@ -183,29 +191,39 @@ VALUES
 Notice that the last partition value `'2021-04-01'` has to match the `TIMESTAMP` 
 date during insertion. There is no validation in Hive to make sure this is 
 happening because it only requires a `VARCHAR` and knows to partition based on 
-different values. Likewise, if a user runs this DML statement, they get the
-correct results back, but have to scan all the data in the process as they
-forgot to include the `event_time_day < '2021-04-02'` predicate in the `WHERE` 
-clause. This eliminates all the benefits that led us to create the partition in
-the first place.
+different values. 
+
+On the other hand, if a user runs the following query, they get the
+correct results back, but have to scan all the data in the table.
```

Comment:
Maybe try:

`If a user runs the following query:`

And then move the `they get the correct results back` to where you have `Result:` below.